tron/bin/check_job_exceeding_time.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3.8
+import logging
+import sys
+
+import pytimeparse
+
+from tron.commands import cmd_utils
+from tron.commands.client import Client
+
+
+log = logging.getLogger("check_exceeding_time")
+
+
+def parse_cli():
```

Comment:
and depending on how easy it is to trace the tron code to get the type of some of these things, it might be worth using `reveal_type` (see https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#when-you-re-puzzled-or-when-things-are-complicated) to have mypy tell you what it thinks the types are